File: sort/k번째수.js
function solution(array, commands) {
  let answer = [];
  let length = commands.length;
  for (let t = 0; t < length; t++) {
    let [i, j, k] = commands[t];
    let tmp = array.slice(i - 1, j);
    tmp = tmp.sort((a, b) => a - b);
    answer.push(tmp[k - 1]);
  }
  return answer;
}

File: sort/k번째수.java
import java.util.Arrays;

class Solution {
    public int[] solution(int[] array, int[][] commands) {
        int n = commands.length;
        int[] answer = new int[n];
        
        int[] command;
        int[] newCommand;
        
        for (int i = 0; i < n; i++) {
            command = commands[i];
            newCommand = Arrays.copyOfRange(array, command[0] - 1, command[1]);
            Arrays.sort(newCommand);
            answer[i] = newCommand[command[2] - 1];
        }
        return answer;
    }
}
